NuRadioReco/modules/channelGalacticNoiseAdder.py

```diff
@@ -50,32 +72,101 @@ def begin(
         ----------
         debug: bool, default: False
             It True, debug plots will be shown
+        skymodel: {'lfmap', 'lfss', 'gsm2016', 'haslam', 'ssm', 'gmoss', 'ulsa_fdi', 'ulsa_dpi', 'ulsa_ci'}, optional
+            Choose the sky model to use. If none is provided, the Global Sky Model (2008) is used as a default.
         n_side: int, default: 4
             The n_side parameter of the healpix map. Has to be power of 2
             The radio skymap is downsized to the resolution specified by the n_side
             parameter and for every pixel above the horizon the radio noise coming
             from that direction is calculated. The number of pixels used is
             12 * n_side ** 2, so a larger value for n_side will result better accuracy
             but also greatly increase computing time.
-        interpolation_frequencies: array of float
+        freq_range: array of len=2, default: [30, 80] * units.MHZ
             The sky brightness temperature will be evaluated for the frequencies
-            in this list. Brightness temperature for frequencies in between are
+            within this limit. Brightness temperature for frequencies in between are
             calculated by interpolation the log10 of the temperature
             The interpolation_frequencies have to cover the entire passband
             specified in the run method.
```

Comment:
...alternatively, we can keep the argument `interpolation_frequencies`. Your call.

Reply:
Good suggestion, I kept both and print a warning when interpolation_frequencies is used. 
I much prefer freq_range since it does not require the user to set up the frequencies themselves, especially since log interpolation seems like a more ideal choice for Galactic Noise.

Reply:
But it also does not allow the user to set the frequency spacing themself since the number of bins is fixed. I would keep both - why deprecating one?